example/lib/main.dart

```diff
@@ -40,102 +52,265 @@ class _MyAppState extends State<MyApp> {
 
   @override
   Widget build(BuildContext context) {
+
     return MaterialApp(
         home: Scaffold(
       appBar: AppBar(
         title: const Text('flutter_radar_example'),
       ),
-      body: Container(
-        child: Column(children: [
-          Permissions(),
-          TrackOnce(),
-          RaisedButton(
-            color: Colors.blueAccent,
-            onPressed: () {
-              Radar.startTracking('responsive');
-            },
-            child: Text('startTracking(\'responsive\')'),
-          ),
-          RaisedButton(
-            color: Colors.blueAccent,
-            onPressed: () {
-              Radar.startTrackingCustom({
-                'desiredStoppedUpdateInterval': 60,
-                'fastestStoppedUpdateInterval': 60,
-                'desiredMovingUpdateInterval': 30,
-                'fastestMovingUpdateInterval': 30,
-                'desiredSyncInterval': 20,
-                'desiredAccuracy': 'high',
-                'stopDuration': 140,
-                'stopDistance': 70,
-                'sync': 'all',
-                'replay': 'none',
-                'showBlueBar': true
-              });
-            },
-            child: Text('startTrackingCustom()'),
-          ),
-          RaisedButton(
-            color: Colors.blueAccent,
-            onPressed: () {
-              Radar.stopTracking();
-            },
-            child: Text('stopTracking()'),
-          ),
-          RaisedButton(
-            color: Colors.blueAccent,
-            onPressed: () {
-              Radar.mockTracking(
-                  origin: {'latitude': 40.78382, 'longitude': -73.97536},
-                  destination: {'latitude': 40.70390, 'longitude': -73.98670},
+      body: SingleChildScrollView(
+        scrollDirection: Axis.vertical,
+        child: Container(
+          child: Column(children: [
+            Permissions(),
+            TrackOnce(),
+            ElevatedButton(
+              style: raisedButtonStyle,
+              onPressed: () async {
+                var resp = await Radar.startTrip({
+                    "externalId": '299',
+                    "destinationGeofenceTag": 'store',
+                    "destinationGeofenceExternalId": '123',
+                    "mode": 'car',
+                    "scheduledArrivalAt": "2020-08-20T10:30:55.837Z",
+                    "metadata": {"test": 123}
+                    },
```

Comment:
@gabrielalao These should be nested under `tripOptions`

Reply:
fixed

Reply:
fixed